d2l-offscreen-shared-styles.html

```diff
@@ -12,7 +12,7 @@
 					left: -10000px;
 				};
 			}
-			:host-context([dir="rtl"]) {
+			:host(:dir(rtl)) {
```

Comment:
Hmm... I'm really surprised by this (thought we would have tested shady A LOT). Also, caniuse says the `:dir` pseudo class shouldn't be working in Chrome/IE/Edge/Safari:
http://caniuse.com/#feat=css-dir-pseudo